index.html

```diff
@@ -63,7 +63,7 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
   <input type="checkbox" value="cityM">City Meetups</input><br>
   <input type="checkbox" value="wiki">Wiki</input><br>
   <input type="checkbox" value="forum">Forum</input><br>
-  <input type="checkbox" value="additional courses">Additional Courses</input><br>
+  <input type="checkbox" value="ad courses">Additional Courses</input><br>
   <div>
```

Comment:
Great job arranging your codes with <div>s, But I would suggest using semantic tags instead to increase efficiency of screen readers

Reply:
- [ ] Your review is well given, you are polite and all but it does not make any sense at all, how are you requesting the student to use the semantic tag on the `input` element ? what do you mean by that? please make sure you give an accurate review comment, you have explained the `why`, and now please explain `what` the student should change, not only a vague review, @David-Lanzz